js/components/operatorSignIn/operatorSignInModal.tsx

```diff
@@ -37,17 +35,20 @@ enum CompleteState {
 const submit = (
   badgeEntry: BadgeEntry,
   radio: string,
-  expired: Certification[],
-  missing: MissingCertification[],
+  certificationStatus: CertificationStatus | null,
```

Comment:
This feels a bit weird. If certificationStatus is null (cuz it's loading), then you shouldn't submit, that makes sense. What do you think about making it the caller's responsibility to not call `submit()` in that case instead of taking the null and doing nothing with it? That might make it more likely that the caller remembers to have a disabled button or something.